Apps/Laboratory/src/Models/PHSAResult.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Laboratory.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// Representation of the result sent by PHSA.
+    /// </summary>
+    public class PHSAResult
+    {
+        /// <summary>
+        /// Gets or sets the result section.
+        /// </summary>
+        [JsonPropertyName("result")]
+        public List<LaboratoryOrder>? Result { get; set; }
```

Comment:
The IEnumerable wouldn't work here right (because of deserialize?)?

Reply:
I think so since the json object is an array